openshift3/persistent-postgresql.json

```diff
@@ -27,7 +27,7 @@
         {
             "name": "WEBLATE_VOLUME_SIZE",
             "description": "Size of the persistent volume for Weblate.",
-            "value": "1Gi",
+            "value": "40Gi",
```

Comment:
Are these changes intentional? It heavily increases storage requirements...

Reply:
Yes. `1gi` is OK for initial setup, but if you work on a project, that is bigger then 1GB (which is sadly today in `node` world quite often), this will obviously not work.

And 40GB today is really not a lot.

What do you think?

Reply:
I have no clue what impact this has on OpenShift - force example if you can override that somehow or whether it just makes running Weblate unconditionally expensive. My guess is that most users won't need that much of space.

Reply:
Fair enough, will lower it to 10Gi.

Reply:
The whole point of it being a template parameter though is that people can override it when instantiating the template. In that sense, is better to set it to small value so test installs will work, with people who are doing production installs overriding it to what they believe they need. The alternative is to leave the field empty value, but with the field being required. This forces everyone to specify size as template parameter.

Setting allow expansion as default for PVC will also break for most people as not typical that OpenShift clusters would allow that. So deploy would presumably fail.

Key with the template is that it really should only serve as an example. For production use people should be copying it and customising it to their needs. So I would suggest leave it as 1Gi and not enable volume expansion.

Similar problem with memory. Now will not work for test installs.

My benchmark would be can it be installed in OpenShift Online starter tier.

Reply:
The allow expansion could be a template parameter also with default of false.

Reply:
@GrahamDumpleton OK, lowered it to 10Gi, i think this is reasonable. Also set `allowVolumeExpansion` to false.

Would that work?